docs/UserGuide.md

```diff
@@ -74,6 +74,7 @@ FINDvisor is a **desktop app for financial advisors to manage contacts, optimize
 
 **:information_source: Notes about input values format:**<br>
 
+* Input values cannot be blank unless specified by feature description.
```

Comment:
```suggestion
* Input values cannot be blank unless otherwise specified.
```
Would this be better to keep our descriptions more consistent?

Reply:
yes I agree.